src/main/kotlin/gg/ingot/iron/representation/EntityField.kt

```diff
@@ -14,5 +15,6 @@ internal data class EntityField(
     val javaField: Field,
     val columnName: String,
     val nullable: Boolean,
-    val isJson: Boolean
+    val isJson: Boolean,
+    val deserializer: ColumnDeserializer<*, *>?
```

Comment:
We may want to specify a type generic on the deserializer based on the type generic on the class (which currently doesn't exist)

Reply:
Invalid request.